### PR TITLE
update build scripts and readme

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -133,6 +133,9 @@ Currently, the publishing process is manual:
 ----
 
 . Review the output (`build\site\index.html`) for errors. Correct if required.
++
+NOTE: Check for correct styling changes too.
```
Comment (finp) [inline README.adoc]: +1
I think, over time, we'll add to this, ie a list of checks

**Comment by pb82** [comment]
With this change it's best to use the antora cli directly to build the local site: `antora --clean --pull local-site.yml`

**Comment by finp** [comment]
@pb82  running antora, rather than quick-build.sh, won't render the docs correctly, not sure what your comment relates to?

**Comment by pb82** [comment]
@finp build.sh used to build `local-site.yml` and i was using that to test local changes. But looks like i can use `quick-build.sh` too.

**Pull request merged**